strings/CMakeLists.txt

```diff
@@ -41,7 +41,9 @@ ADD_EXECUTABLE(conf_to_src EXCLUDE_FROM_ALL conf_to_src.c)
 SET_TARGET_PROPERTIES(conf_to_src PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD TRUE)
 TARGET_LINK_LIBRARIES(conf_to_src mysys strings)
 
-ADD_EXECUTABLE(uca-dump uca-dump.c)
+IF(NOT CMAKE_CROSSCOMPILING OR DEFINED CMAKE_CROSSCOMPILING_EMULATOR)
+  ADD_EXECUTABLE(uca-dump uca-dump.c)
```

Comment:
Quick question: doesn't this completely disable the `uca-dump` from compiling when cross compiling? If so, isn't this needed for generating `ctype-uca1400data.h` which is needed for compiling?

Reply:
Never mind, answered my own question when I figured out the EXPORT part of the CMake.